bot/exts/utils/color.py

```diff
@@ -0,0 +1,64 @@
+from io import BytesIO
+from string import hexdigits
+
+from PIL import Image
+from bot.bot import Bot
+from discord import File
+from discord.ext.commands import Cog, Context, command
+
+
+class Color(Cog):
```

Comment:
```suggestion
class ColorVisualizer(Cog):
```
Seems a more appropriate name.

Reply:
Maybe. It's longer and to be frank, it feels like bikeshedding at this point. Let's just use `Color`.

Reply:
`Color` is the name of our class in constants storing the colours, so it is better to keep it more specific.